tests/acceptance/pageObjects/sharedWithMePage.js

```diff
@@ -16,38 +16,35 @@ module.exports = {
       )
     },
     /**
+     * Checks if the file-row of the desired file-name with the username is present with the desired status of accepted,
+     * declined or pending
+     *
      * @param {string} filename
      * @param {string} status - It takes one of the following : declined, pending or '' for accepted
      * @param {string} user
-     * Checks if the file-row of the desired file-name with the username consists of the desired status of accepted,
-     * declined or pending
+     *
+     * @return {Promise<boolean>}
      */
-    assertDesiredStatusIsPresent: function (filename, status, user) {
+    isDesiredStatusPresent: async function (filename, status, user) {
```

Comment:
the logic about comparing the status is still in this function. Simply retrieve the status with a function like `getSharingStatus` and make all the decisions, about what is the desired status in the context file

Reply:
done!